app/controllers/api/quality/similarity_job.rb

```diff
@@ -0,0 +1,143 @@
+class SimilarityJob
+
+  JAVA           = '/usr/bin/java'
+  BED_DIR        = '/usr/bin/'
+  RSCRIPT        = '/usr/bin/Rscript'
+  SIMILARITY_DIR = '/opt/genetalk/addons/similarity/'
```

Comment:
For these path 'opt/genetalk/...', we should put in config.yaml. Please check this file https://github.com/igsb/dpdl/blob/dev/docs/PEDIA.md and https://github.com/igsb/dpdl/blob/dev/app/models/pedia_service.rb from line 34.

I separate the production and development environment. For the path in development environment, developer should put the path of all the external tools in config.yaml. We should avoid to modify the code for local testing, even though this file is from Tom.

Please also check this https://stackoverflow.com/questions/13310488/how-can-i-read-a-yaml-file for parsing config from yaml file.